src/cvodes/cvodes.c

```diff
@@ -8517,13 +8563,14 @@ static int cvQuadEwtSetSS(CVodeMem cv_mem, N_Vector qcur, N_Vector weightQ)
 
 static int cvQuadEwtSetSV(CVodeMem cv_mem, N_Vector qcur, N_Vector weightQ)
 {
-  N_VAbs(qcur, cv_mem->cv_tempvQ);
+  SUNAssignSUNCTX(CV_SUNCTX);
+  SUNCheckCallLastErrNoRet(N_VAbs(qcur, cv_mem->cv_tempvQ));
   N_VLinearSum(cv_mem->cv_reltolQ, cv_mem->cv_tempvQ, ONE,
                cv_mem->cv_VabstolQ, cv_mem->cv_tempvQ);
   if (cv_mem->cv_atolQmin0) {
     if (N_VMin(cv_mem->cv_tempvQ) <= ZERO) return(-1);
```

Comment:
As above, needs a `sunrealtype minval` declared at the top of the function, and then this should be changed to something like
```suggestion
    minval = SUNCheckCallLastErrNoRet(N_VMin(cv_mem->cv_tempvQ));
    if (minval <= ZERO) return(-1);
```